src/InstrumentationEngine.Api/InstrumentationEngine.idl

```diff
@@ -462,6 +463,39 @@ library MicrosoftInstrumentationEngine
         HRESULT GetInstructionGeneration([out] enum InstructionGeneration* pInstructionGeneration);
     };
 
+    // Specialization of IInstruction, useful for fast looping.
+    [
+        object,
+        uuid(A77D20D8-B261-41C7-9B0A-F5FBBAABA576),
+        pointer_default(unique)
+    ]
+    interface IInstruction2 : IInstruction
+    {
+        // Returns the next instruction without incrementing the reference count
```

Comment:
This there a way to get these comments into the header file? I don't know if anyone outside of us will understand the intent of these methods.

Reply:
I can't find a way to do that, and there aren't any examples of that that I can find, I'm afraid.